### DISCUSSION TITLE
Resetting the 'id' field, how to?

### DISCUSSION THREAD

**Discussion by Ryuk-hash** [strapi/strapi#6148] [Legacy Topics]
A little help, please? I've just started exploring strapi. ;)

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by lauriejim** [comment]
Hello! Can you please precise you question.
I don't understand what you are trying to do.

**Reply by ghost** [reply]
I don't know if understand clearly. 

For example, I created a category type with news, interview and editorial, and also the ID 1, 2, 3 respectively. If I delete the news category afterwards, and create a new one called video with the ID 4. Now I get IDs from 2 - 4. But what I want to get is ID 1-3.

**Reply by ghost** [reply]
And I found that even firstly deleted the content type and re-created it afterwards, the IDs would not be reset also.

**Reply by Ryuk-hash** [reply]
> I don't know if understand clearly.
> 
> For example, I created a category type with news, interview and editorial, and also the ID 1, 2, 3 respectively. If I delete the news category afterwards, and create a new one called video with the ID 4. Now I get IDs from 2 - 4. But what I want to get is ID 1-3.

Exactly, what I wanted to ask. Sorry to not explain much more or either reply - totally lost track of this thread as I didn't receive a reply for so long.

**Reply by Ryuk-hash** [reply]
I think there's a scope for a minor feature here.

**Reply by ajj-siraj** [reply]
I want to do this as well. It's pretty annoying to see so many gaps in the ID numbers. I understand not changing IDs for existing entries but why does it not 'reset' to the earliest available ID for a NEW entry? IDs don't need to be in chronological order, that's what "created_at" is for isn't it? Is there a way to do this manually?

**Reply by derrickmehaffy** [reply]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
